lib/escher.js

```diff
@@ -11,11 +11,20 @@ var Escher = function(configToMerge) {
     algoPrefix: 'ESR',
     vendorKey: 'ESCHER',
     hashAlgo: 'SHA256',
-    date: new Date(),
     credentialScope: 'escher_request',
     authHeaderName: 'X-Escher-Auth',
     dateHeaderName: 'X-Escher-Date',
-    clockSkew: 300
+    clockSkew: 300,
+    get date() {
+      return this._date || new Date();
+    },
+    set date(date) {
+      if (!(configToMerge.date instanceof Date)) {
```

Comment:
Are you sure you wanted to write configToMerge.date here instead of date?

Reply:
Right, it should be `date`. Fixed it, thank you!